packages/lib-classifier/src/components/Classifier/components/MetaTools/MetaTools.stories.js

```diff
@@ -0,0 +1,77 @@
+import zooTheme from '@zooniverse/grommet-theme'
+import { Box, Grommet } from 'grommet'
+import { Provider } from 'mobx-react'
+import {
+  DrawingTaskFactory,
+  SubjectFactory,
+  UPPFactory,
+  WorkflowFactory
+} from '@test/factories'
+import mockStore from '@test/mockStore'
+
+import MetaTools from './MetaTools'
+
+export default {
+  title: 'Meta Tools / MetaToolsContainer',
+  component: MetaTools,
+  args: {
+    dark: false
+  }
+}
+
+const background = {
+  dark: 'dark-1',
+  light: 'light-1'
+}
+
+const mockSubject = SubjectFactory.build({
+  locations: [
+    {
+      'image/jpeg':
+        'https://panoptes-uploads.zooniverse.org/subject_location/1e54b552-4608-4701-9db9-b8342b81278a.jpeg'
+    }
+  ],
+  metadata: {
+    time: 'Morning',
+    location: 'United States'
+  }
+})
+
+const mockWorkflow = WorkflowFactory.build({
+  tasks: {
+    T0: DrawingTaskFactory.build()
+  }
+})
+
+const upp = UPPFactory.build()
+
+const store = mockStore({
+  subject: mockSubject,
+  workflow: mockWorkflow
+})
+
+store.userProjectPreferences.setUPP(upp)
+
+export const Default = ({ dark }) => {
+  const themeMode = dark ? 'dark' : 'light'
+  return (
+    <Grommet background={background} theme={zooTheme} themeMode={themeMode}>
+      <Provider classifierStore={store}>
+        <MetaTools />
+      </Provider>
+    </Grommet>
+  )
+}
+
+export const SmallSubjectViewer = ({ dark }) => {
+  const themeMode = dark ? 'dark' : 'light'
+  return (
+    <Grommet background={background} theme={zooTheme} themeMode={themeMode}>
+      <Provider classifierStore={store}>
+        <Box width='300px' style={{ border: 'solid 1px red' }}>
```

Comment:
I'm not sure what this shows, maybe a comment to help clarify what the red box represents and how the MetaTools component should look or change from the default? 

If the idea is to show what the classifier looks like on a small screen, I changed the layout to an iPhone12 in the storybook and the buttons looked good and as expected. I'm not sure if it's possible, but maybe changing the defaultLayout just for this story to an iPhone or mobile dimensions might be helpful?

Reply:
I think the border was left over from building the story, since it just makes the containing `div` easier to see. The idea of this story is to show what the Classifier's subject grid area looks like when its containing `div` is small. Remember that this case does not occur only on mobile screens. The default layout is 2-column, and both columns shrink in width as a browser window decreases in width.

I removed the border.